.github/workflows/release.yaml

```
@@ -60,13 +75,23 @@ jobs:
           username: chrislusf
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
-      - name: Build
+      - name: Build csi-driver image
         # if: ${{ startsWith(github.ref, 'refs/tags/') }}
         uses: docker/build-push-action@v2
```

actionlint (1.7.9)

the runner of "docker/build-push-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)